docs/help/operator.md

```diff
@@ -737,6 +737,28 @@ spec:
       promethes-prop: "somevalue"
 ```
 
+### Custom Labels and Annotations on supporting resources; Services, Ingress, Secrets etc.
+It is possible to configure a Template(s) for resources that are managed by the operator.
+The Template contains a Labels and Annotations with an optional Selector. If the selector is empty
+the template matches all resources. Othewise it can be used to restrict what is matched.
+Note: the relevant variables supported by `ingressHost` can be referenced in keys and values for both labels and annotations.
```

Comment:
IngressHost seems confusing as there is nowhere else it is mentioned.

Reply:
It it would be nice to document supported resource types for labels and annotations. (i.e. services, secrets, ingresses/routes)

Reply:
thanks, I wonder if it is trivial to link to that?

The reference to ingressHost is via the CRD: https://github.com/artemiscloud/activemq-artemis-operator/blob/main/deploy/crds/broker_activemqartemis_crd.yaml#L59

Reply:
yeah, that'll be great

Reply:
https://github.com/artemiscloud/activemq-artemis-operator/issues/614 I will link to the issue